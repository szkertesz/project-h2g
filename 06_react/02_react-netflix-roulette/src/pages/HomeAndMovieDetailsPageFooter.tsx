import Container from '../ui/Container';
import Logo from '../assets/images/logo_netflixroulette.svg';
import classes from './HomeAndMovieDetailsPageFooter.module.scss';

function HomeAndMovieDetailsPageFooter() {
    return (
        <footer className={classes.footer}>
            <Container className={classes['footer__inner']}>
                <img src={Logo} alt='logo' className={classes['footer__logo']} />
            </Container>
        </footer>
    );
}

export default HomeAndMovieDetailsPageFooter;
