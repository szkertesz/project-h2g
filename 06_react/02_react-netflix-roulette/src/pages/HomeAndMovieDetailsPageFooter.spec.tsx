import HomeAndMovieDetailsPageFooter from "./HomeAndMovieDetailsPageFooter";
import renderer, { ReactTestRenderer } from 'react-test-renderer'

describe('Footer', () => {
    let component: ReactTestRenderer;

    beforeEach(() => {
        component = renderer.create(
            <HomeAndMovieDetailsPageFooter />
        )
    })

    it('should render component', () => {
        expect(component.toJSON()).toMatchSnapshot();
    })
})