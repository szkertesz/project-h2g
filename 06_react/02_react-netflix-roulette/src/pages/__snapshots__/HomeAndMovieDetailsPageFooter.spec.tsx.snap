// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render component 1`] = `
<footer
  className="footer"
>
  <div
    className="container footer__inner"
  >
    <img
      alt="logo"
      className="footer__logo"
      src="logo_netflixroulette.svg"
    />
  </div>
</footer>
`;
